MerlinAU.sh

```diff
@@ -4610,15 +4614,15 @@ _SelectOfflineZipFile_()
     do
         printf "\nAvailable ZIP files in the directory: [${GRNct}${FW_ZIP_DIR}${NOct}]:\n\n"
 
-        zipFileList="$(ls -A1 "$FW_ZIP_DIR"/*.zip 2>/dev/null)"
+        fileList="$(ls -A "$FW_ZIP_DIR"/*.w "$FW_ZIP_DIR"/*.pkgtb "$FW_ZIP_DIR"/*.zip 2>/dev/null)"
```

Comment:
>
> The instructions will say a ZIP, if they drop a raw file it won't work since it's expecting a zip for the rest of the update to work. (It would just fail at the next step further down).
>
Line **4617** is including raw image files in the "**filelist**" which is then used to display the file selection menu to the user. If only ZIP files will be accepted in some cases, there's no conditional logic to indicate that. Showing ZIP files along with raw image files for selection would be confusing when **only** ZIP files are valid selections.

Am I understanding your intent correctly??

Reply:
I see your point.

I was just going to let it fail if they didn't follow the steps as it was written lol. I was going to let it accept anything. That way no need for additional conditional statements for Gnuton, the one function can take raw or zip.

But just adjust the wording for Gnuton vs Merlin. And let it fail id they didn't follow it.

Aka it would display both. But if they dropped a .w file when it asked for a .zip further down it will fail with the update run saying the .zip isn't found. I wasnt going to baby anymore than that lol

Reply:
Undone. Will do conditionals for the Gnuton branch.